easybuild/easyconfigs/c/CRPropa/CRPropa-3.1.5-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
```

Comment:
This is probably easier to implement as a CMakePythonPackage.
You can then remove the modextrapaths and the ".../site-packages" from sanity_check.

Reply:
done.